### PR TITLE
[RO] Add `Help centre`

### PR DESCRIPTION
**Pull request by 28edx**
Translation for the `Help centre` article.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/ro.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Secțiuni {id=sections}
+
+Alege secțiunea care se potrivește cel mai bine cu problema ta pentru a găsi o soluție potrivită cu această problemă.
+
+| Secțiune | Zonă de interes |
+| :-- | :-- |
+| [Cont](/wiki/Help_centre/Account) | suporter osu!, autentificare și accesare recuperare, modificări nume, date profil |
+| [Restricții cont](/wiki/Help_centre/Account_restrictions) | Totul despre restricții, prezentare generală, proceduri de apel, motive comune și cooldown |
+| [Beatmapping și Editor](/wiki/Help_centre/Beatmapping) | Păstrarea beatmap-urilor tale, proprietatea unui beatmap, slot-uri beatmap |
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:24]:
```suggestion
| [Beatmapping și Editor](/wiki/Help_centre/Beatmapping) | Păstrarea beatmap-urilor tale, proprietatea unui beatmap, sloturi beatmap |
```

File: wiki/Help_centre/ro.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+**Fișierele jurnal sunt înregistrări detaliate despre ce face client-ul jocului în orice moment dat. Le putem folosi pentru a determina cauza problemei tale.**
+
+Aceste fișiere sunt extrem de utile și pot face rezolvarea chiar și a problemelor foarte complicate să fie trivială.
+
+Dacă un membru din echipa de asistență ți-a cerut aceste jurnale, uite cum poți face rost de ele:
+
+1. Deschide osu!.
+2. Apasă pe butonul `Opțiuni` în meniul principal sau apasă `Ctrl` + `O`.
+3. Scrie `release` în bara de căutare rapidă. Asta te va duce la linia de release curentă folosită de client-ul tău.
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:44]:
```suggestion
3. Scrie `release` în bara de căutare rapidă. Asta te va duce la canalul de distribuție actual folosit de client-ul tău.
```
Can potentially update to match https://crowdin.com/editor/osu-web/529/en-ro?view=comfortable#16593

File: wiki/Help_centre/ro.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+| [Actualizarea la lazer](/wiki/Help_centre/Upgrading_to_lazer) | Migrarea la [următoarea versiune semnificativă](/wiki/Client/Release_stream/Lazer) osu! |
+| [Site](/wiki/Help_centre/Website) | Blocharea utilizatorilor, contactarea suportului, aparența site-ului |
+
+## Ajută-ne să te ajutăm {id=diagnostics}
+
+### Fișiere jurnal {id=log-files}
+
+**Fișierele jurnal sunt înregistrări detaliate despre ce face client-ul jocului în orice moment dat. Le putem folosi pentru a determina cauza problemei tale.**
+
+Aceste fișiere sunt extrem de utile și pot face rezolvarea chiar și a problemelor foarte complicate să fie trivială.
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:38]:
```suggestion
Aceste fișiere sunt extrem de utile și pot face rezolvarea chiar și a celor mai complicate probleme să fie trivială.
```

File: wiki/Help_centre/ro.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+3. Scrie `release` în bara de căutare rapidă. Asta te va duce la linia de release curentă folosită de client-ul tău.
+4. Asigură-te că este setată pe `Experimental`.
+5. Apasă pe butonul de restart care apare în capătul ecranului dacă ai făcut modificări la linia de release pentru a le aplica.
+6. Du-te din nou în meniul de `Opțiuni` și apasă pe `Deschide directorul osu!`
+7. Găsește directorul `Logs` în fereastra care se deschide.
+8. Selectează fișierul de jurnal necesar (reprezenantul de asistență îți va spune care este) și atașează-l la tichetul de asistență sau la postarea de pe forum.
+
+### Event Viewer {id=event-viewer}
+
+**Event Viewer-ul este un instrument incorporat în Windows care poate fi utilizat pentru a găsi jurnalele cu crash-uri când osu! nu ți le poate oferi. Aceste jurnale cu crash-uri pot fi folosite pentru a determina ce cauzează problema ta.**
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:53]:
```suggestion
**Event Viewer-ul este un instrument incorporat în Windows care poate fi utilizat pentru a găsi jurnalele cu crash-uri când osu! nu ți le poate oferi. Aceste jurnale pot fi folosite pentru a determina cauza problemei tale.**
```

File: wiki/Help_centre/ro.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+1. Deschide osu!.
+2. Apasă pe butonul `Opțiuni` în meniul principal sau apasă `Ctrl` + `O`.
+3. Scrie `release` în bara de căutare rapidă. Asta te va duce la linia de release curentă folosită de client-ul tău.
+4. Asigură-te că este setată pe `Experimental`.
+5. Apasă pe butonul de restart care apare în capătul ecranului dacă ai făcut modificări la linia de release pentru a le aplica.
+6. Du-te din nou în meniul de `Opțiuni` și apasă pe `Deschide directorul osu!`
+7. Găsește directorul `Logs` în fereastra care se deschide.
+8. Selectează fișierul de jurnal necesar (reprezenantul de asistență îți va spune care este) și atașează-l la tichetul de asistență sau la postarea de pe forum.
+
+### Event Viewer {id=event-viewer}
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:51]:
```suggestion
### Vizualizator Evenimente {id=event-viewer}
```
If we are translating the name for the run box, this should also be translated (usage should be updated to match)

Source: https://support.microsoft.com/ro-ro/topic/vizualizator-evenimente-se-poate-%C3%AEnchide-sau-poate-primi-o-eroare-atunci-c%C3%A2nd-utiliza%C8%9Bi-vizualiz%C4%83rile-particularizate-e44da633-a81e-e554-9abf-ec70caae879c

File: wiki/Help_centre/ro.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+**Event Viewer-ul este un instrument incorporat în Windows care poate fi utilizat pentru a găsi jurnalele cu crash-uri când osu! nu ți le poate oferi. Aceste jurnale cu crash-uri pot fi folosite pentru a determina ce cauzează problema ta.**
+
+Atunci când sunt probleme cu osu! crash, dacă osu! nu vă oferă un fișier jurnal cu crash-ul, singurul loc unde poți găsi unul este în Event Viewer.
+
+Dacă un membru din echipa de asistență ți-a cerut să cauți un fișier jurnal de crash în Event Viewer, uite cum poți face asta:
+
+1. După ce osu! a dat crash, apasă `Win` + `R` pentru a deschide caseta de Executare.
+2. În caseta de Executare scrie `eventvwr` și apasă `Enter`. Asta va deschide Event Viewer.
+3. În Event Viewer, în stânga, apasă pe `Windows Logs` și apoi `Application`.
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:61]: Translate as mentioned above or keep it all in English

File: wiki/Help_centre/ro.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+4. În dreapta, apasă pe `Filter current log`.
+5. În fereastra de filtrare care s-a deschis, asigură-te că ai căsuța de `Error` bifată și apasă `OK`.
+6. Apasă `Ctrl` + `F` și scrie osu! în căsuța de căutare. O să găsească primul jurnal cu osu! crash.
+7. Du-te în fila de `Details`, extinde `System` și `Event Data` făcând clic pe ele.
+8. Copiază textul de acolo și lipește-l în tichetul de asistență sau în postarea de pe forum.
+
+### Grafic de timp-cadre {id=frame-time-graph}
```
Comment (NiceAesth) [inline wiki/Help_centre/ro.md:68]:
```suggestion
### Grafic de timp cadre {id=frame-time-graph}
```

**Review by NiceAesth — CHANGES_REQUESTED** [review]
translation looks ok otherwise, just these mentions

File: wiki/Help_centre/ro.md (hunk 5 above)
Comment (28edx) [reply wiki/Help_centre/ro.md:51]: Event Viewer is not localized while the run box is. I checked it. You still have to type "Event Viewer" in windows to search it.

**Review by NiceAesth — APPROVED** [review]
looks ok now

**Review by TicClick — APPROVED** [review]

**Pull request merged**